pkgs/tools/security/tessen/default.nix

```
@@ -0,0 +1,46 @@
+{ lib
+, stdenvNoCC
+, fetchFromSourcehut
+, pass
+, bemenu
+, passBackend ? pass
+, picker ? bemenu
+, wtype
+, wl-clipboard
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "tessen";
+  version = "2.1.2";
+
+  src = fetchFromSourcehut {
+    owner = "~ayushnix";
+    repo  = pname;
+    rev = "8758a09345f6eef24764de4a0efad737f12562c8";
+    sha256  = "sha256-U6obXpYzIprOJ+b3QiE+eDOq1s0DYiwM55qTga9/8TE=";
+  };
+
+  buildInputs = [ passBackend picker wtype wl-clipboard ];
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/bin
+    mkdir -p $out/share/man
+    mkdir -p $out/etc/xdg/tessen
+
+    cp -a tessen $out/bin/tessen
+    cp -r man $out/share/man/tessen
+    cp -a config $out/etc/xdg/tessen
+
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    homepage = "https://sr.ht/~ayushnix/tessen";
+    description = "An interactive menu to autotype and copy Pass and GoPass data";
+    license = licenses.gpl2;
```

```suggestion
    license = licenses.gpl2Plus;
```
based on https://git.sr.ht/~ayushnix/tessen/tree/master/item/LICENSE#L299